spec/acceptance/golden_masters_spec.rb

```diff
@@ -7,9 +7,10 @@
 
   after do
     verify do
-      DB[:estimates].all.map do |row|
-        [[row[:subject_id], row[:user_id], row[:answer], row[:probability].round(20)]]
-      end
+      DB[:estimates].all.map { |row| row[:data] = JSON.load(row[:data]); row }
+                        .flat_map { |row| (row[:data]["guesses"] || []).map {|guess| guess.merge(row) } }
+                        .sort_by { |row| row["timestamp"] }
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 18) spaces for indenting an expression spanning multiple lines.